selfdrive/ui/qt/maps/map.cc

```diff
@@ -319,6 +320,7 @@ void MapWindow::offroadTransition(bool offroad) {
 }
 
 MapInstructions::MapInstructions(QWidget * parent) : QWidget(parent) {
+  is_rhd = Params().getBool("IsRHD");
```

Comment:
can we get this from mapbox instead?

Reply:
Hi @adeebshihadeh,
thanks! according to the [direction api](https://docs.mapbox.com/api/navigation/directions/):

`driving_side` column is `The legal driving side at the location for this step. Either left or right.`

however, it is under `routes[0] > legs[0] > steps[0] > driving_side`
 
![image](https://user-images.githubusercontent.com/16603033/141424424-b7488269-076a-49ae-aec3-cdac906d3e1d.png)

any idea where I will have access to that? looks like `cur_maneuver.extendedAttributes()` does not have it.

Reply:
The parser from the qtlocation plugin is not very complete. It's on my own list to write my own parser so we can also get speed limits properly.

Reply:
So we can use the RHD toggle/param for now and switch to this output once we have our own parser.